src/CoreWCF.Primitives/src/CoreWCF/Dispatcher/SecurityReplyChannelBinder.cs

```diff
@@ -0,0 +1,202 @@
+﻿using CoreWCF.Channels;
+using CoreWCF.Configuration;
+using CoreWCF.Description;
+using CoreWCF.Diagnostics;
+using CoreWCF.Runtime;
+using CoreWCF.Security;
+using CoreWCF.Security.Tokens;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Xml;
+
+namespace CoreWCF.Dispatcher
+{
+    //For Security over SOAP, creating a separte Binder
```

Comment:
The entire implementation in this file has been commented out. Remove file as nothing left is used.

Reply:
so many files to port, while we are working through various design in mind, sometime I miss these kind of unnecessary file(necessary file becoming unnecessary).. so thanks for pointing it out..